e2e/app-with-domain/app_with_domain_suite_test.go

```diff
@@ -33,7 +33,7 @@ var _ = BeforeSuite(func() {
 	copilotCLI, err := client.NewCLI()
 	cli = copilotCLI
 	Expect(err).NotTo(HaveOccurred())
-	appName = fmt.Sprintf("e2e-domain-%d", time.Now().Unix())
+	appName = fmt.Sprint(time.Now().Unix())
```

Comment:
I guess no longer needed because of the separate accounts/test?

Reply:
that is right. Also another important motivation was that, with `e2e-domain-xxx`, the certificate that we request per env would be: `test.e2e-domain-xxxxxx.app-with-domain.copilot.e2e.dev.test` which exceeds 60 characters limit from ACM